server/tests/search_providers/belga_360_archive_test.py

```diff
@@ -378,3 +378,31 @@ def test_get_image_renditions(self):
                 },
             },
         )
+
+    def test_get_highlighted_text(self):
+        with HTTMock(archive_mock):
+            query = {
+                "size": 50,
+                "from": 50,
+                "query": {
+                    "filtered": {
+                        "query": {
+                            "query_string": {"query": "Lorem ipsum"},
```

Comment:
can you pls add a test case where it will be the other way around

Reply:
Yep sure